src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -708,7 +710,23 @@ public class SimpleEvents {
 					.since("2.7")
 					.requiredPlugins("Paper 1.16+");
 		}
-
+		Skript.registerEvent("Entity Breed", SimpleEvent.class, EntityBreedEvent.class, "[entity] breed[ing]")
+			.description("Called whenever two breedable entities begin to conceive a child.")
+			.examples(
+				"on breeding:",
+				"\tsend \"When a %breeding mother% and %breeding father% love each other they make %offspring%\" to breeder"
+			)
+			.since("INSERT VERSION");
+		if (Skript.classExists("org.bukkit.event.entity.EntityEnterLoveModeEvent")) {
+			Skript.registerEvent("Love Mode Enter", SimpleEvent.class, EntityEnterLoveModeEvent.class, "[entity] enter[s] love mode", "[entity] love mode [enter]")
+				.description("Called whenever an entity enters a state of being in love.")
+				.examples(
+					"on love mode enter:",
+					"\tcancel event # No one is allowed love here"
+				)
+				.since("INSERT VERSION")
+				.requiredPlugins("MC 1.16+");
```

Comment:
```suggestion
					.description("Called whenever an entity enters a state of being in love.")
					.examples(
						"on love mode enter:",
							"\tcancel event # No one is allowed love here"
					)
					.since("INSERT VERSION")
					.requiredPlugins("MC 1.16+");
```